lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -165,6 +186,21 @@ def pull_tests_from_queue(can_initialize_queue: false)
           test_file_paths
         end
 
+        def print_current_rspec_batch_command
+          test_file_paths = @queue.current_batch&.test_file_paths
```

Comment:
When is the `current_batch` nil? Is it when the trap happened before/in between/after a batch?

Reply:
Yes. Trap could happen before tests are pulled from the Queue API.